data_dir/cs_cdc_simple_write.yaml

```diff
@@ -0,0 +1,41 @@
+keyspace: cdc_keyspace
+
```

Comment:
This file cs_cdc_simple_write.yaml is added, but really used.

Reply:
it could be used for test_write_profile method

Reply:
please don't add files that are not used. put it in a separated PR with the code that use that